sentry_sdk/scope.py

```diff
@@ -244,11 +245,22 @@ def get_dynamic_sampling_context(self):
 
         return self._propagation_context["dynamic_sampling_context"]
 
-    def get_traceparent(self):
-        # type: () -> Optional[str]
+    def get_traceparent(self, *args, **kwargs):
+        # type: (Any, Any) -> Optional[str]
         """
-        Returns the Sentry "sentry-trace" header (aka the traceparent) from the Propagation Context.
+        Returns the Sentry "sentry-trace" header (aka the traceparent) from the
+        currently active span or the scopes Propagation Context.
         """
+        client = kwargs.pop("client", None)
+
+        # If we have an active span, return trashparent from there
```

Comment:
```suggestion
        # If we have an active span, return traceparent from there
```
I do like `trashparent` as a name though 😀